src/python/pants/binaries/binary_tool.py

```diff
@@ -190,6 +191,17 @@ class Script(BinaryToolBase):
   """
   platform_dependent = False
 
+  def hackily_snapshot(self, context):
+    bootstrapdir = self.__class__.global_instance().get_options().pants_bootstrapdir
```

Comment:
A constructed instance should have options as well... so `self.get_options()` should work...?